policyengine_us/variables/gov/states/hi/tax/income/exemptions/hi_exemptions.py

```diff
@@ -0,0 +1,44 @@
+from policyengine_us.model_api import *
+
+
+class hi_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Hawaii exemptions"
+    unit = USD
+    documentation = (
+        "https://files.hawaii.gov/tax/forms/2022/n11ins.pdf#page=20"
+    )
+    definition_period = YEAR
+    defined_for = StateCode.HI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.hi.tax.income.exemptions
+        exemp = tax_unit("exemptions", period)
+        # disability exemption
+        disabled_head = tax_unit("head_is_disabled", period).astype(int)
```

Comment:
And elsewhere; we only need this if we're adding them together to get numbers like 1 + 1 = 2. Here we're just multiplying them so bools suffice.
```suggestion
        disabled_head = tax_unit("head_is_disabled", period)
```